src/ConfigR/Scripting/ScriptConfigLoader.cs

```diff
@@ -64,25 +64,25 @@ private static void RethrowExceptionIfAny(ScriptResult result, string scriptPath
         {
             if (result.CompileExceptionInfo != null)
             {
-                result.CompileExceptionInfo.Throw();
-            }
-
-            if (result.ExecuteExceptionInfo != null)
-            {
                 // HACK: waiting on https://github.com/scriptcs/scriptcs/issues/545

```

Comment:
this is changed because Roslyn 1.0.0-rc2 doesn't throw execution exception anymore for empty files